### DISCUSSION TITLE
How to add an OrgContact

### DISCUSSION THREAD

**Discussion by D8planet** [microsoftgraph/msgraph-sdk-dotnet#2438] [Q&A] [answered]
Basically i was trying to create a simple app in C# to sync user contacts so that they can be seen in the Default Global Address List (GAL), and found myself using graph.

When i use `await graphClient.Contacts.PostAsync(OrgContact newContact);` 
the error ` Unsupported resource type 'OrgContact' for operation 'Create'.`  pops up.

I am new to C# programming, so it could very well be my fault but everything else works (getting the full contact list works and i can connect to the Exchange Online Server).

I have seen that i could create a new user with a mailbox, but it would defeat the purpose of having contacts, i just want them to be visible in the GAL.
I just wanted to have some clarifications because i'm lost; also if there's  a better approach i'm open to suggestions.
Thanks in advance

**Comment by D8planet** [comment]
Some more context if needed: the idea is to take all contacts from my personal Contacts folder (that i can see in Outlook for example) and make those contacts visible in the GAL of my organization.
I know that i can create a contact from the [Exchange admin center](https://admin.exchange.microsoft.com/#/contacts) but i wanted to add them dynamically and make the process automatic, like maybe sync them once a week.

**Comment by andrueastman** [comment, marked as answer]
Thanks for raising this @D8planet 

According to the rest Api documentation, the organizational contact is managed by the admin and an operation to create on via the APImay not be supported at the moment. 
https://learn.microsoft.com/en-us/graph/api/resources/orgcontact?view=graph-rest-1.0

**Reply by D8planet** [reply]
Thanks for the quick reply!
Guess i'll have to search a bit more, assuming there is a way to do what i want (i am still kind of lost ngl)

**Comment by MrM40** [comment]
That's are really strange argument!
It's exactly the reason for why it should be possible. It is admins who need to automate/optimate tasks (i.e. using powershell / graph / api etc), the end users don't use, and have no need for that.
